pineappl_py/src/bin.rs

```diff
@@ -6,7 +6,7 @@ use pyo3::prelude::*;
 /// PyO3 wrapper to :rustdoc:`pineappl::bin::BinRemapper <bin/struct.BinRemapper.html>`
 ///
 /// **Usage**: `yadism`
-#[pyclass]
+#[pyclass(name = "BinRemapper")]
```

Comment:
Can't we name the class `BinRemapper` directly?

Reply:
since in Rust one can do `use pineappl::bin::BinRemapper as RustBinRemapper` I guess it should be possible. We were most likely influenced by the PyO3 naming scheme, but there is no need to follow them - things are scoped by crate. However, note that this might trigger more changes (in number of changed characters) than the present version.

Reply:
I don't have a preference for this. Adding custom names (for what are exposed) required the lowest amount of changes while still complying with the PyO3 naming scheme, but if you really want to change these I think we can do so.

Reply:
Actually, in hindsight, I think it is preferable to keep it this way - this is exactly the way PyO3 recommends.